### PR TITLE
[C-2971] Add Avatar

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds basic Avatar component, which will be used to improve seo by properly wrapping user images in an anchor tag with proper accessibility

Also adds improved useUserProfilePicture hook which only needs user_id. kinda lame name but eventually we can migrate to this

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-avatar

File: packages/web/src/components/avatar/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { SIGN_IN_PAGE, profilePage } from 'utils/route'
+
+import styles from './Avatar.module.css'
+
+const { getAccountUser } = accountSelectors
+
+const { getUser } = cacheUsersSelectors
+
+const messages = {
+  goTo: 'Go to',
```
Comment (raymondjacobson) [inline packages/web/src/components/avatar/Avatar.tsx:22]: is `go to` std verbiage?

File: packages/web/src/hooks/useUserProfilePicture.ts [hunk 2]
```diff
@@ -28,6 +32,27 @@ export const useUserProfilePicture = (
   })
 }
 
+export const useUserProfilePicture2 = (
```
Comment (raymondjacobson) [inline packages/web/src/hooks/useUserProfilePicture.ts:35]: maybe a more descriptive name or a comment suggesting the different args

**Review by raymondjacobson — APPROVED** [review]

File: packages/web/src/components/avatar/Avatar.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/components/avatar/Avatar.tsx:22]: From chat gpt perspective yeah, but can dig into whether there is better verbiage

File: packages/web/src/hooks/useUserProfilePicture.ts (hunk 2 above)
Comment (dylanjeffers) [reply packages/web/src/hooks/useUserProfilePicture.ts:35]: > maybe a more descriptive name or a comment suggesting the different args

Yeah agreed, may also deprecate the old one

**Pull request merged**